src/api/app/models/project.rb

```diff
@@ -968,6 +976,44 @@ def branch_to_repositories_from(project, pkg_to_enable, opts = {})
     end
   end
 
+  def branch_remote_repositories(project)
```

Comment:
I would say that it belongs to project... :thinking: 

We could consider moving it afterwards anyway